### DISCUSSION TITLE
Practica 1 - Abogabot

### DISCUSSION THREAD

**Discussion by LaMazorca** [Launch-X-Latam/MisionFrontEnd#211] [Show and tell]
Hola amigos, quiero compartir lo que llevo de la practica 1, un poco tarde pero ahí vamos, el tiempo que he tenido no ha sido mucho; me gustaría si me pudieran comentar y las criticas constructivas, es la primera vez que realizo algo así.
De ante mano agradezco su ayuda

[Abogabot](https://github.com/LaMazorca/Practica-Abogabot)

**Comment by AlexNasu** [comment]
Se ve muy bien amigo!

**Comment by DiegoCorchuelo** [comment]
Esta Excelente, Te felicito!
